src/pages/home-page.js

```diff
@@ -110,6 +110,26 @@ const HomePage = (props) => {
               CSElectives team is also recruiting! If you&#39;re interested in the development of this project, projects
               subcom applications close 16th Jan. More details can be found
               <a href="https://fb.me/e/3r4RVffai" target="_blank" rel="noreferrer"> here</a>.
+              !\
+              <pre>             H|H|H|H|H           H__________________________________</pre>
```

Comment:
I think we could probably put this all inside a single `<pre>` element rather than having a separate one for each line, as `<pre>` elements respect newlines. This would also solve the issue where we `1em` margin space between each line due to built-in browser styles, without having to override those.

However, since we are using React with JSX, simple new lines will be treated as spaces (see https://github.com/facebook/react/issues/13781), meaning we'll have to use a JS expression, perhaps with an extracted variable or with an inline template literal, to include line breaks. Moreover, to keep the formatting the same with a template literal, we're forced to unindent everything to the leftmost column, as the leading whitespace will be rendered. However, there's also the choice of creating an array of strings for each line that we then `.join('\n')`, but this has its own downsides. In any case, the code style may be affected and thus this will have to be considered with deep thought.

I think we should either choose an option for keeping it in one `<pre>` element, or fix the styles to remove margins between the lines to cleanly render this beautiful American flag.